xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-notifications/src/main/resources/templates/notification/rss/mentions.mention.vm

```diff
@@ -56,8 +56,11 @@
             #set ($lastIndex = $event.events.size() - 1)
             #set ($origdoc = $xwiki.getDocument($event.document, $event.events[$lastIndex].documentVersion).previousDocument)
         #end
-        #set ($externalURL = $newdoc.getExternalURL('view', "viewer=changes&rev1=${origdoc.version}&rev2=${newdoc.version}"))
-        #set ($diffLink = "<a href='$externalURL'>${newdoc.version}</a>")
-        $services.localization.render('notifications.rss.seeChanges', 'xwiki/2.1', [$diffLink])
+        #if ($newdoc.version)
```

Comment:
Those changes are not related to message stream: should be part of another commit I guess.

Reply:
Changes removed and moved to https://jira.xwiki.org/browse/XWIKI-17923 and https://jira.xwiki.org/browse/XWIKI-17924